schematron/v2.1.0/L000_Prelim_Analysis.csv

```diff
@@ -0,0 +1,21 @@
+phase title,phase see,pattern title,pattern see,rule title,rule context,assert test,assert description,assert severity,notes
+Preliminary Analysis,ASHRAE 211 5.2.3,Document Structure,,,/,count(auc:BuildingSync) = 1,,,
+,,,,,/auc:BuildingSync,count(auc:Facilities) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities,count(auc:Facility) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility,count(auc:Sites) = 1,,,
+,,,,,,count(auc:Reports) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites,count(auc:Site) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site,count(auc:Buildings) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings,count(auc:Building) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports,count(auc:Report) >= 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report,count(auc:Scenarios) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios,count(auc:Scenario) >= 1,,,

```

Comment:
Shouldn't we have >= 2 scenarios? i.e. a measured and benchmark?

Reply:
Resolved by generating schematron tests for rule contexts